### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cevro**

### CODE REVIEW

**Comment by rzehumat** [comment]
Are you going to commit sth more, or can we `Update branch` and `Merge`?

File: README.md [hunk 1]
```diff
@@ -1,39 +1,88 @@
...
+
 Initialization
 --------------
 
-0.1) Checkout branch 'dev'
+0.1) Checkout branch `master`
 
 0.2) Run `git submodule init && git submodule update`
 
+0.3) Run `copmposer install`
```
Comment (rzehumat) [inline README.md:37]: Isn't it rather `composer install` (missclicked letter `p`)?

Comment (cevro) [reply README.md]: Thx, fixed.

**Comment by cevro** [comment]
Mergnem to a ak bude treba niečo spraviť spravíme nový PR.

**Pull request merged**